complete-application/mysite/mysite/settings.py

```diff
@@ -34,6 +34,7 @@
 OIDC_OP_JWKS_ENDPOINT = os.environ['ISSUER'] + "/.well-known/jwks.json"
 LOGIN_REDIRECT_URL = "http://localhost:8000/app/account/"
 LOGOUT_REDIRECT_URL = "http://localhost:8000/app/"
+# LOGOUT_REDIRECT_URL = os.environ['ISSUER'] + "/oauth2?client_id=" + OIDC_RP_CLIENT_ID
```

Comment:
Logout redirect should be: `LOGOUT_REDIRECT_URL = os.environ['ISSUER'] + "/oauth2/logout?client_id=" + OIDC_RP_CLIENT_ID`